pkg/network/vmicontroller/status_test.go

```diff
@@ -45,6 +46,29 @@ var _ = Describe("Status Update", func() {
 
 		multusNetworkStatusWithPrimaryNet = `[{"name":"k8s-pod-network","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}}]`
 
+		multusNetworkStatusWithPrimaryNetAndIfaceName = `[` +
+			`{"name":"k8s-pod-network","interface":"eth0","ips":["10.244.196.146","fd10:244::c491"],"default":true,"dns":{}}` +
+			`]`
+
+		multusNetworkStatusWithAlternativePrimaryNet = `[` +
+			`{"name":"alternativeNAD", "ips":["10.244.196.146", "fd10:244::c491"], "default":true, "dns":{}}` +
+			`]`
+
+		multusNetworkStatusWithAlternativePrimaryNetAndIfaceName = `[` +
+			`{"name":"alternativeNAD", "interface":"eth0", "ips":["10.244.196.146", "fd10:244::c491"], "default":true,"dns":{}}` +
+			`]`
+
+		multusNetworkStatusWithCustomPrimaryNet = `[` +
+			`{"name":"k8s-pod-network", "interface":"eth0", "ips":["10.244.196.146", "fd10:244::c491"], "default":false,"dns":{}}, ` +
+			`{"name":"cluster-network", "interface":"custom-iface", "ips":["10.128.0.4"], "mac":"0a:58:0a:80:00:04", "default":true, "dns":{}}` +
+			`]`
+
+		multusNetworkStatusWithCustomPrimaryAndSecondaryNets = `[` +
+			`{"name":"k8s-pod-network", "interface":"eth0", "ips":["10.244.196.146", "fd10:244::c491"], "default":false,"dns":{}}, ` +
+			`{"name":"cluster-network", "interface":"custom-iface", "ips":["10.128.0.4"], "mac":"0a:58:0a:80:00:04", "default":true, "dns":{}}, ` +
+			`{"name":"meganet", "interface":"pod7e0055a6880", "mac":"8a:37:d9:e7:0f:18", "dns":{}}` +
+			`]`
+
```

Comment:
nit: looking at these tests again, I think it would make it easier to review and follow if these const were inlined.